### PR TITLE
docs(web-modeler): switch to SBOM dependency list

### PR DESCRIPTION
**Pull request by wollefitz**
## Description

- Closes https://github.com/camunda/web-modeler/issues/4386
- Relates to https://github.com/camunda/camunda-docs/pull/4277 and https://github.com/camunda/engineering-leadership-team/issues/68

## When should this change go live?

- [ ] This is a bug fix, security concern, or something that needs **urgent release support**.
- [ ] This is already available but undocumented and should be released within a week.
- [ ] This on a **specific schedule** and the assignee will coordinate a release with the DevEx team. (apply `hold` label or convert to draft PR)
- [x] This is part of a scheduled alpha or minor. (apply alpha or minor label)
- [ ] There is **no urgency** with this change and can be released at any time.

## PR Checklist

- [x] My changes are for an **already released minor** and are in `/versioned_docs` directory.
- [x] My changes are for the **next minor** and are in `/docs` directory (aka `/next/`).

- [x] My changes require an [Engineering review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned the Engineering DRI or delegate.
- [x] My changes require a [technical writer review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned @camunda/tech-writers as a reviewer.

### CODE REVIEW

**Comment by wollefitz** [comment]
Hey @MaxTru and/or @kevinCamunda!

Could you please have a look at the proposed wording of the Web Modeler dependencies section meets our requirements?

> - **Dependencies:** SBOM CycloneDX files with up-to-date lists of third party libraries used and their licenses can be requested on demand.
> - **Source code:** Source code access can be provided on demand.

Do we have to mention SaaS and SM (and their individual components `webapp`, `restapi` and `websockets`) separately or is this sufficient?

File: docs/reference/dependencies.md [hunk 1]
```diff
@@ -3156,1011 +3156,8 @@ All of these libraries are required for core functionality.
...
-- [symfony/translation@v7.1.3](https://github.com/symfony/translation.git) (MIT)
-- [symfony/translation-contracts@v3.5.0](https://github.com/symfony/translation-contracts.git) (MIT)
-- [symfony/uid@v7.1.1](https://github.com/symfony/uid.git) (MIT)
-- [symfony/var-dumper@v7.1.3](https://github.com/symfony/var-dumper.git) (MIT)
-- [tijsverkoyen/css-to-inline-styles@v2.2.7](https://github.com/tijsverkoyen/CssToInlineStyles.git) (BSD-3-Clause)
-- [vlucas/phpdotenv@v5.6.1](https://github.com/vlucas/phpdotenv.git) (BSD-3-Clause)
-- [voku/portable-ascii@2.0.1](https://github.com/voku/portable-ascii.git) (MIT)
-- [webmozart/assert@1.11.0](https://github.com/webmozarts/assert.git) (MIT)
+- **Dependencies:** SBOM CycloneDX files with up-to-date lists of third party libraries used and their licenses can be requested on demand.
+- **Source code:** Source code access can be provided on demand.
```
Comment (MaxTru) [inline docs/reference/dependencies.md:3160]:
```suggestion
- **Source code:** Access to source code access is provided on [on-demand](dependency-request@camunda.com).
```

**Review by MaxTru — CHANGES_REQUESTED** [review]

**Comment by MaxTru** [comment]
Hi @wollefitz ,

thanks - looks good! Just please include the email address already.

Note: I will go over the detailed wording once all components have done their part (ensuring consistency and legal aligned wording).

File: docs/reference/dependencies.md (hunk 1 above)
Comment (wollefitz) [reply docs/reference/dependencies.md]: ✅ Added an email address link to both bullet points.

File: docs/reference/dependencies.md [hunk 2]
```diff
@@ -3156,1011 +3156,9 @@ All of these libraries are required for core functionality.
 
 <TabItem value='web-modeler'>
 
-### Web Modeler Dependencies (webapp)
+- **Dependencies:** SBOM CycloneDX files with up-to-date lists of third party libraries used and their licenses can be requested [on-demand](mailto:dependency-request@camunda.com).
+- **Source code:** Access to source code is provided on [on-demand](mailto:dependency-request@camunda.com).
```
Comment (MaxTru) [inline docs/reference/dependencies.md:3160]:
```suggestion
- **Source code:** Access to source code is provided [on-demand](mailto:dependency-request@camunda.com).
```

**Review by MaxTru — CHANGES_REQUESTED** [review]
Just a small detail wording

File: docs/reference/dependencies.md (hunk 2 above)
Comment (wollefitz) [reply docs/reference/dependencies.md]: Thanks and sorry for the oversight!

**Comment by wollefitz** [comment]
Additionally requesting a review from @camunda/tech-writers as I'm a bit confused by "on demand" vs "on-demand" - maybe you could assist me 🙂

**Comment by christinaausley** [comment]
> Additionally requesting a review from @camunda/tech-writers as I'm a bit confused by "on demand" vs "on-demand" - maybe you could assist me 🙂

"On demand" is correct here. You might use on-demand as an adjective preceding a noun to describe the noun -- for example, on-demand television".

**Review by christinaausley — APPROVED** [review]

**Comment by wollefitz** [comment]
Thanks for having another look! Let's merge this then 🚀

**Pull request merged**